src/components/withKeyboardState.tsx

```diff
@@ -40,14 +47,9 @@ function KeyboardStateProvider(props) {
     return <KeyboardStateContext.Provider value={contextValue}>{children}</KeyboardStateContext.Provider>;
 }
 
-KeyboardStateProvider.propTypes = keyboardStateProviderPropTypes;
-
-/**
- * @param {React.Component} WrappedComponent
- * @returns {React.Component}
- */
-export default function withKeyboardState(WrappedComponent) {
-    const WithKeyboardState = forwardRef((props, ref) => (
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export default function withKeyboardState(WrappedComponent: ComponentType<{ref: ForwardedRef<unknown>}>) {
```

Comment:
For HOC typing, please follow the logic other HOCs use. Take a look at `withLocalize` HOC as a reference.
1. Add HOC return type
2. Use generics
3. Update forwardRef logic